docs/book/src/introduction.md

```diff
@@ -1 +1,91 @@
-# Introduction
+# Kubernetes Cluster API Provider IBM Cloud
+
+<p align="center">
+<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100x">
+</p>
+
+------
+<p align="center">Kubernetes-native declarative infrastructure for IBM Cloud.</p>
+
+## What is the Cluster API Provider IBM Cloud
+
+The [Cluster API](https://github.com/kubernetes-sigs/cluster-api) brings declarative, Kubernetes-style APIs to cluster creation, configuration and management.
+
+The API itself is shared across multiple cloud providers allowing for true IBM Cloud
+hybrid deployments of Kubernetes.  It is built atop the lessons learned from
+previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
+[kubicorn](http://kubicorn.io/).
+
+## Quick Start
+
+Check out the [Cluster API Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.html) to create your first Kubernetes cluster on IBM Cloud using Cluster API.
```

Comment:
I feel the mentioned link contains the overall information about cluster-api and doesn't contain any information about the IBM Cloud, so wondering how this information will be useful for the user?

Reply:
Since the cluster-api quick start page consists of instructions to deploy a workload cluster on most of the infrastructure providers, this was added with an intention to have IBM Cloud added on their page. For now, I have changed this to our getting-started page.